private/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
This file allows an empty ./private folder in this repo, but will ignore any file in it. The purpose is to create a local `seed.json` file that is used to store wallet JSON but not upload to Github.

Reply:
It's OK!
But I think dotenv-cli is also good.

Reply:
Yeah, feel free to make it work with `.env`, but I think in order to save the json in .env you need to convert it to a string.